### DISCUSSION TITLE
ASUS VG27AQA1A and LG 27QN600-B

### DISCUSSION THREAD

**Discussion by jeshadr** [xanderfrangos/twinkle-tray#776] [Monitor Compatibility Issues]
I have two monitors, ASUS VG27AQA1A and LG 27QN600-B. They are both connect to a RTX 4070 using DisplayPort cable. Both monitors have DDC/CI and is enabled on both. Previously I was able to use Twinkle Tray to control my monitors brightness but I uninstalled it and reinstalled it and now it does not detect any of my monitors. I tried Monitorian and am able to control both my monitors brightnesses.

**Comment by Cloweeee** [comment]
I have an ASUS VG27AQL1A (slightly different) and it's also not getting detected. Other programs are able to.
